src/interfaces/interfaces.tsx

```diff
@@ -0,0 +1,7 @@
+export interface ICard {
```

Comment:
interfaces/interfaces is very interesting path)) better create file with name that describes entity that is inside file